src/app/public/plugin-resources/code-examples/definition-list/basic/definition-list-demo.component.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  Component
+} from '@angular/core';
+
+@Component({
+  selector: 'app-definition-list-demo',
+  templateUrl: './definition-list-demo.component.html'
+})
+export class DefinitionListDemoComponent {
+
+  public items: { label: string, value: string }[] = [
+    {
+      label: 'Field 1',
+      value: 'Field 1 value'
+    },
+    {
+      label: 'Field 2',
+      value: 'Field 2 value'
+    },
+    {
+      label: 'Field 3',
+      value: undefined
```

Comment:
Why are we setting this to `undefined`?

Reply:
So it shows in the demo as "none found". Definition list has the functionality built in, and this attempts to show it off.